app/controllers/alchemy/pages_controller.rb

```diff
@@ -104,7 +104,7 @@ def load_index_page
     # @return NilClass
     #
     def load_page
-      @page ||= Page.contentpages.find_by(
+      @page ||= Language.current.pages.find_by(
```

Comment:
Imo in the API this is not the case, right? We want to be able to load a global page via the API. The current behaviour is that we load a page by urlname, whatever type of page it is.